api/src/stampy_chat/chat.py

```diff
@@ -0,0 +1,247 @@
+# ------------------------------- env, constants -------------------------------
+from dataclasses import asdict
+from typing import List, Dict, Callable
+import openai
+import re
+import tiktoken
+import time
+
+from stampy_chat.followups import multisearch_authored
+from stampy_chat.get_blocks import get_top_k_blocks, Block
```

Comment:
these two lines are the only change (prepended `stampy_chat.` and moved them lower, as local imports should be last)